.gitmodules

```diff
@@ -50,6 +50,8 @@
 	path = third_party/cirque/repo
 	url = https://github.com/openweave/cirque.git
 	branch = master
+	ignore = dirty
+	commit = 905112cbeeda504223209798ec23e3754ce43fca
```

Comment:
after you pushed 0.4.10 chip-build-cirque image to dockerhub, you might have the issue that I encountered before, which is cirque unable to pull the latest code. It keeps on using old commit id.

Reply:
do you have a workaround or diagnosis of the issue you had?